test/kuzzle-sdk-test.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * This file only purpose is to check that type remains compliant with the spec for user usage.
+ * This is not meant to be executed. It is only a compilation check along the test battery.
+ *
+ * If you see red warning, it's likely that you broke the code somewhere and introduced a breaking change.
+ *
+ * TODO: This can be safely removed when test will be migrated to TypeScript.
+ *
+ * Inspired by TS standard from DefinitelyTyped
+ * @see https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs
+ */
+import { Kuzzle } from "../src/Kuzzle";
+import WebSocket from "../src/protocols/WebSocket";
+const kuzzle = new Kuzzle(new WebSocket("toto"));
+
+// Events
+kuzzle.on("connected", () => {});
+kuzzle.on("callbackError", ({ error, notification }) => {});
+kuzzle.on(
+  "discarded",
+  ({
+    action,
+    controller,
+    _id,
+    body,
+    collection,
+    index,
+    jwt,
+    requestId,
+    volatile,
+  }) => {}
+);
+kuzzle.on("disconnect", ({ origin }) => {});
```

Comment:
This is going to throw since the SDK does not know the `disconnect` event, only `disconnected`